docs/release-process.md

```diff
@@ -19,6 +19,27 @@ build/Dockerfile.release
 go.mod
 ```
 
+### Update Compatibility Table
+
+Add a new entry for the planned minor version to the [compatibility table](https://github.com/planetscale/vitess-operator/blob/main/README.md#compatibility)
+in the README file.
+
+The recommended Kubernetes versions depend on the version of the Kubernetes
+client libs that Vitess Operator uses:
+
+https://github.com/planetscale/vitess-operator/blob/111ac173e1c473853e66e270486ba8a9a47ecc54/go.mod#L35
```

Comment:
Do we want this to be a permanent/static link?

Reply:
I think it is fine to keep it that way. The link redirects to an old commit but at least it will always point to the same line of code. That line:
```
// Pinned to kubernetes-1.16.2
```

Is visible in all the versions of our `go.mod`, with the `1.16.2` being the only thing we change.